### PR TITLE
GH-2: Replaces `context` with `contextId`.

### LINKED ISSUE #2: Replace `context` with `contextId` in the keybindings
So that it is aligned with the most recent Theia API.
See also: https://github.com/theia-ide/theia/issues/914

AC:
 - Bump up version.
 - Clean up `package.json`. One should be able to install, build and run the tests.
 - Release a new version.

### PR DESCRIPTION
**Pull request by kittaakos**
So that it is aligned with the most recent (0.2.3) Theia API. Updated `packaged.json` to have a self-contained npm package.

Closes #2.

Signed-off-by: Akos Kitta <kittaakos@gmail.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ bundle.js
 bundle.js.map
 npm-debug.log
 yarn-error.log
+yarn.lock
```
Comment (JanKoehnlein) [inline .gitignore]: We don't ignore that file in other repos, e.g. theia, so why here?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+yarn build
+```
+
+If you would like to run the `tsc` in a watch mode, run the following script:
+
+```bash
+yarn build -- -w
+```
+
+### Testing
```
Comment (JanKoehnlein) [inline README.md]: The only test is pretty bogus. We used it to spot some alleged DI issue, which turned out to be a missing pair of parentheses in our code. I'd rather delete it and not add the entire test infrastructure

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "theia-sprotty",
-  "version": "0.1.5",
+  "version": "0.1.6",
```
Comment (JanKoehnlein) [inline package.json]: Why did you increase the version number? This is usually done by npm when publishing.

File: package.json [hunk 4]
```diff
@@ -22,12 +22,14 @@
   "scripts": {
     "prepare": "yarn run clean && yarn run build",
     "clean": "rimraf lib",
-    "lint": "tslint -c ../tslint.json --project ./tsconfig.json",
-    "build": "tsc && yarn lint",
-    "watch": "tsc -w",
-    "test": "mocha --opts ./mocha.opts"
+    "build": "tsc",
```
Comment (JanKoehnlein) [inline package.json]: Same as above, we should rather get rid of the bogus test

File: README.md (hunk 2 above)
Comment (kittaakos) [reply README.md]: You mean deleting the test module itself?

**Comment by kittaakos** [comment]
I have removed all the unnecessary changes.

**Review by JanKoehnlein — APPROVED** [review]

**Pull request merged**